src/test/java/seedu/address/model/contactedinfo/ContactedInfoTest.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.contactedinfo;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CONTACTED_DATE_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CONTACTED_DATE_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CONTACTED_DESC_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CONTACTED_DESC_BOB;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class ContactedInfoTest {
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new ContactedInfo(null, "meetup"));
+        assertThrows(NullPointerException.class, () -> new ContactedInfo("2020-02-02", null));
+        assertThrows(NullPointerException.class, () -> new ContactedInfo(null, null));
+    }
+
+    @Test
+    public void constructor_invalidDescription_throwsIllegalArgumentException() {
+        String description = "";
+        assertThrows(IllegalArgumentException.class, () -> new ContactedInfo("2020-02-02", description));
+    }
+
+    @Test
+    public void constructor_invalidDate_throwsIllegalArgumentException() {
+        String date = "not a date";
+        assertThrows(IllegalArgumentException.class, () -> new ContactedInfo(date, "meet up"));
```

Comment:
`IllegalArgumentExceptions` are correct, at first I thought we should throw `IllegalValueExceptions` but this is after you parse so `IllegalArgumentException` is more appropriate